superset/templates/appbuilder/navbar_right.html

```diff
@@ -17,8 +17,11 @@
   under the License.
 #}
 
-{% set bug_report_url = appbuilder.app.config['BUG_REPORT_URL'] %}
-{% set documentation_url = appbuilder.app.config['DOCUMENTATION_URL'] %}
+{% set bug_report_url = appbuilder.app.config.get('BUG_REPORT_URL') %}
+{% set documentation_url = appbuilder.app.config.get('DOCUMENTATION_URL') %}
+{% set documentation_icon = appbuilder.app.config.get('DOCUMENTATION_ICON') %}
+{% set documentation_text = appbuilder.app.config.get('DOCUMENTATION_TEXT', 'Documentation') %}
```

Comment:
instead of this, let's default the config var value in `config.py` and keep the `[]` syntax for getting params from this dict. Same goes for all the other config vars here.

Reply:
+1

Reply:
Done. One issue with this is that if `DOCUMENTATION_TEXT` isn't defined in the an existing deployment's config.py then the default text will now be `None`.  Any concerns with that?

Reply:
I think it's fine. The issue would really be if `DOCUMENTATION_TEXT` was explicitly defined as `None`, because otherwise it's now defined in the default config